### ISSUE TITLE
Older project expecting pre-1.0.0 version fails to build due to missing dependencies

### ISSUE THREAD

**Issue by gerardo-rodriguez** [cloudfour/eslint-config#4]
I am not 100% sure how to label this but I wanted to document it somewhere.

I am working on a project that was relying on the pre-1.0.0 version of `cloudfour/eslint-config`. As part of the deployment process for this application to Pivotal Cloud Foundry, all of the dependencies are installed fresh on PCF on each deploy. 

During the last deploy, the build failed because as of `cloudfour/eslint-config` version `1.0.0` there are new dependencies that must be manually installed. I was about to add those dependencies but also realized that the whole codebase for this project was going to run into linting issues (specifically thinking of spaces after function names). It felt like too big of a task to deal with in the moment.

To allow the project to build correctly and continue with our deploy, I instead set the `cloudfour/eslint-config` dependency to point to a specific commit hash (pre-1.0.0).

```diff
"devDependencies": {
-    "@cloudfour/eslint-config": "github:cloudfour/eslint-config",
+    "@cloudfour/eslint-config": "github:cloudfour/eslint-config#0abf408460dc218b1eb825c69476328ed18eaba4"
}
```

As stated earlier, I am not sure this is an _issue_ per say, instead more of a heads up in case any other projects run into this same thing.

**Comment by tylersticka** [comment]
I ran into this yesterday but did not broadcast it (my apologies). I arrived at the same solution.

We should always avoid linking to a dependency with no version specified, and in the future we can reference this config's npm package instead of a commit hash: https://www.npmjs.com/package/@cloudfour/eslint-config

**Comment by calebeby** [comment]
If you use only eslint, no prettier, there won't be any linting errors related to code style

**Comment by gerardo-rodriguez** [comment]
This has now been resolved by publishing an older [pre-`1.0.0` version](https://www.npmjs.com/package/@cloudfour/eslint-config/v/0.1.0) of `@cloudfour/eslint-config` to NPM. I am no longer having any issues. 

My `package.json` diff looks like this now:
```diff
-    "@cloudfour/eslint-config": "github:cloudfour/eslint-config",
+    "@cloudfour/eslint-config": "^0.1.0",
```
